components/script_plugins/lib.rs

```diff
@@ -405,7 +405,11 @@ fn match_def_path(cx: &LateContext, def_id: DefId, path: &[Symbol]) -> bool {
 fn in_derive_expn(span: Span) -> bool {
     matches!(
         span.ctxt().outer_expn_data().kind,
-        ExpnKind::Macro(MacroKind::Derive, _)
+        ExpnKind::Macro {
+            kind: MacroKind::Derive,
+            name: _,
+            proc_macro: _,
```

Comment:
```suggestion
            ..
```

Nit (no need to go back for this if you’re not making other changes): unused fields in patterns can be omitted with `..`